Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "ClimaCore"
 uuid = "d414da3d-4745-48bb-8d80-42e94e092884"
 authors = ["CliMA Contributors <clima-software@caltech.edu>"]
-version = "0.10.25"
+version = "0.10.26"
```

Comment:
Given that we've broken a bunch of stuff, we may need to make the a breaking release.